pkg/propertyprovider/aks/provider.go

```diff
@@ -29,26 +29,16 @@ import (
 )
 
 const (
-	// A list of properties that the AKS property provider collects.
+	// A list of properties that the Azure property provider collects in addition to the
```

Comment:
the package name should be Azure

Reply:
Done. Eventually this package will be moved to its separate repo I hope.